n2y/main.py

```diff
@@ -131,7 +151,7 @@ def export_yaml(client, raw_rows):
         markdown = pandoc.write(pandoc_output, format='gfm') if pandoc_output else None
         result.append({**simplify.flatten_database_row(row), 'content': markdown})
 
-    print(yaml.dump(result, sort_keys=False))
+    logger.info(yaml.dump(result, sort_keys=False))
```

Comment:
Should be a print statement not a logger message. Logging is a side channel. YAML should go to stdout.